packages/ui/src/index.scss

```
@@ -25,6 +25,10 @@
   font-size: 14px;
   accent-color: var(--theme);
 
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
```

Can you check if just setting:
```suggestion
  -webkit-appearance: caret;
```
wouldn't work?
Setting `appearance` to `none` removes the dropdown arrow from all browsers 😞